snippets/oadp-windows-vm-backup-fails.adoc

```diff
@@ -0,0 +1,10 @@
+
+//This snippet appears in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc
+
+:_mod-docs-content-type: SNIPPET
+[WARNING]
+====
+If you run a backup of a Windows VM immediately after the VM reboots, the backup might fail with `WaitingForPluginOperationsPartiallyFailed` error. This is because, in Windows,  there is a hard-coded 10 seconds `filesystem` freeze when a VM boots. Immediately after a VM boots, the Windows Volume Shadow Copy Service (VSS) and Guest Agent (GA) service are not ready. The `filesystem` freeze, and the VSS and GA service not being ready, cause the backup to fail. In such a case, re-try the backup a few minutes after the VM boots.
```

Comment:
```suggestion
If you run a backup of a Microsoft Windows virtual machine (VM) immediately after the VM reboots, the backup might fail with `WaitingForPluginOperationsPartiallyFailed` error. This is because, on Microsoft Windows,  there is a hard-coded 10-second `filesystem` freeze when a VM boots. Immediately after a VM boots, the Microsoft Windows Volume Shadow Copy Service (VSS) and Guest Agent (GA) service are not ready. The `filesystem` freeze, and the VSS and GA service being unready, causes the backup to fail. In such a case, re-try the backup a few minutes after the VM boots.
```